WatchFaceFormat/validator-plugin/plugins/src/main/kotlin/ValidatorExec.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.gradle.api.GradleException
+import org.gradle.api.Project
+import java.nio.file.Path
+
+
+/**
+ * Runs the validator against WFF XML files found in the project structure, matching any files:
+ *
+ *    res\raw*\*.xml
+ */
+internal fun validateWffFiles(project: Project, version: Int) {
+    val resourcePath = Path.of(project.projectDir.absolutePath, "src", "main", "res")
```

Comment:
I suspect this doesn't track the files as inputs correctly, but let me confirm.

Reply:
Sort of this, https://tomgregory.com/gradle/gradle-task-inputs-and-outputs/

Reply:
But maybe a follow up PR?

Reply:
Thanks, have switched to a task with incremental changes